sql/updates/world/3.3.5/2021_80_98_12_world.sql

```diff
@@ -0,0 +1,381 @@
+(33186,0,1,33233,630.2436,-276.2591,392.3122,1.797689,8,0), -- Razorscale Controller
+(33186,0,1,33233,638.2416,-272.1735,392.1351,1.815142,8,0), -- Razorscale Controller
+(33186,0,1,33233,633.6514,-287.3748,391.8054,1.797689,8,0), -- Razorscale Controller
+(33186,0,1,33233,605.9334,-140.0912,391.6004,4.485496,8,0), -- Razorscale Controller
+(33186,0,1,33233,572.4106,-138.6564,393.9044,4.764749,8,0), -- Razorscale Controller
+(33186,0,1,33233,558.9486,-142.9874,391.6004,4.764749,8,0), -- Razorscale Controller
+(33186,0,1,33233,589.7287,-137.1148,393.9011,4.485496,8,0), -- Razorscale Controller
+-- 10 man group
+(33186,0,2,33282,589.6996,-134.6657,391.6004,4.555309,8,0), -- Razorscale Harpoon Fire State
+(33186,0,2,33282,572.0398,-136.2224,391.2637,4.642576,8,0), -- Razorscale Harpoon Fire State
+-- 25 man group
+(33186,0,3,33282,559.5352,-140.9866,391.6004,4.642576,8,0), -- Razorscale Harpoon Fire State
+(33186,0,3,33282,606.2806,-137.2628,391.6004,4.537856,8,0); -- Razorscale Harpoon Fire State
+
+DELETE FROM `gossip_menu_option` WHERE `MenuID`=10314;
+INSERT INTO `gossip_menu_option` (`MenuID`,`OptionID`,`OptionIcon`,`OptionText`,`OptionBroadcastTextID`,`OptionType`,`OptionNpcFlag`,`ActionMenuID`,`ActionPoiID`,`BoxCoded`,`BoxMoney`,`BoxText`,`BoxBroadcastTextID`,`VerifiedBuild`) VALUES
+(10314, 0, 0, 'We are ready to help!', 33353, 1, 1, 0, 0, 0, 0, '', 0, 0);
+
+DELETE FROM `creature_text` WHERE `CreatureID`=33210;
+INSERT INTO `creature_text` (`CreatureID`,`GroupID`,`ID`,`Text`,`Type`,`Language`,`Probability`,`Emote`,`Duration`,`Sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
+(33210,0,0,'Be on the lookout! Mole machines will be surfacing soon with those nasty Iron dwarves aboard!',14,0,100,0,0,0,33607,0,'Expedition Commander SAY_AGGRO'),
+(33210,1,0,'Move quickly! She won\'t remain grounded for long!',14,0,100,0,0,15648,33606,0,'Expedition Commander SAY_GROUND_PHASE'),
+(33210,2,0,'We have lost our engineers, this will not end well!',14,0,100,0,0,0,33818,0,'Expedition Commander SAY_ENGINEERS_DEAD');
+
+DELETE FROM `creature_text` WHERE `CreatureID`=33186 AND `GroupID`=2;
+INSERT INTO `creature_text` (`CreatureID`,`GroupID`,`ID`,`Text`,`Type`,`Language`,`Probability`,`Emote`,`Duration`,`Sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
+(33186,2,0,'%s goes into a berserker rage!',41,0,100,0,0,0,34057,0,'Razorscale EMOTE_BERSERK');
+UPDATE `creature_text` SET `comment`='Razorscale EMOTE_PERMA_GROUND' WHERE `CreatureID`=33186 AND `GroupID`=0 AND `id`=0;
+UPDATE `creature_text` SET `comment`='Expedition Engineer SAY_AGGRO' WHERE `CreatureID`=33287 AND `GroupID`=0;
+UPDATE `creature_text` SET `comment`='Expedition Engineer SAY_START_REPAIR' WHERE `CreatureID`='33287' AND `GroupID`=1;
+UPDATE `creature_text` SET `comment`='Expedition Engineer SAY_REBUILD_TURRETS' WHERE `CreatureID`='33287' AND `GroupID`=2;
+
+UPDATE `creature_template` SET `InhabitType`=5 WHERE `entry`IN (33186, 33724); -- Razorscale
```

Comment:
you set canfly as default and use disablegravity on the script.
first of all hack, secondly like I told you you have to find which of them is the correct. You cant use both, only one active.